endoreg_db/serializers/video/video_file_detail.py

```
@@ -49,10 +37,11 @@ def get_full_path(self, obj):
 
     def get_video_url(self, obj):
         request = self.context.get("request")
+        #TODO remove hardcoded path here 
         return request.build_absolute_uri(f"/api/media/videos/{obj.pk}/") if request else None
     
-    def get_duration(self, obj):
-        return obj.duration or _calc_duration(self, obj)
+    def get_duration(self, obj:VideoFile):
+        return obj.duration or _calc_duration_vf(self, obj)
```

Pylint (3.3.7)

[error] Too many positional arguments for function call

(E1121)